src/04_features/modalMaterial/modalMaterial.html

```diff
@@ -0,0 +1,67 @@
+<!-- modalMaterial.html -->
+<div id="upload-modal" class="upload-modal">
+    <div class="modal-content">
+      <button class="modal-close" id="UploadModal-close">&times;</button>
+      <h2>Страница загрузки материала</h2>
+      <form id="upload-form">
+        <div class="input-group">
+          <label for="name">Имя файла</label>
+          <input type="text" id="name" name="name" required>
+        </div>
+        <div class="input-group">
+          <label for="semester">Семестр (из списка)</label>
+          <select id="semester" name="semester" required>
+            <option value="1">1</option>
+            <option value="2">2</option>
+          </select>
+        </div>
```

Comment:
IF filters selectors from sidebar is not suitable for your case, you need to create new selectors as a independent component. 
I can suggest the following structure:
03_widget
-----modal 
05_entries
----form
------form.js -> for click events and submit
------form.css
------form.html -> includes inputs and selectors from ui kit
06_shared
----ui
-------inputs
-----------textInput
-----------selector
-----------radioInput
-----------fileInput